linux_os/guide/system/software/integrity/software-integrity/aide/aide_periodic_checking_systemd_timer/bash/sle15.sh

```diff
@@ -0,0 +1,34 @@
+# platform = multi_platform_sle
```

Comment:
```suggestion
# platform = multi_platform_all
```
I also suggest renaming the file to shared.sh. It would make the rule adoption easier for other products if necessary in the future.